humbug.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I'm curious as to why it should not be committed ? Isn't it the equivalent of phpunit.xml or any other test config ?